src/gql/queries/get-task-logs.ts

```diff
@@ -1,7 +1,7 @@
 import gql from "graphql-tag";
 
 export const GET_EVENT_LOGS = gql`
-  query taskLogs($id: String!) {
+  query eventLogs($id: String!) {
```

Comment:
CamelCase?

Reply:
Do you mean TitleCase? its already in camelCase.

Reply:
yes TitleCase, thank you :)